modules/timeplus/src/main/java/org/testcontainers/timeplus/TimeplusContainer.java

```diff
@@ -0,0 +1,119 @@
+package org.testcontainers.timeplus;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Testcontainers implementation for Timeplus.
+ */
+public class TimeplusContainer extends JdbcDatabaseContainer<TimeplusContainer> {
```

Comment:
why two implementations? Let's keep this one but please apply the comments I left in the other one

Reply:
The other one is deprecated, i prefer to keep this